src/base/search/searchpluginmanager.cpp

```diff
@@ -637,29 +511,12 @@ bool SearchEngine::isUpdateNeeded(QString pluginName, PluginVersion newVersion)
     return (newVersion > oldVersion);
 }
 
-QString SearchEngine::pluginPath(const QString &name)
+QString SearchPluginManager::pluginPath(const QString &name)
 {
     return QString("%1/%2.py").arg(pluginsLocation()).arg(name);
 }
 
-QHash<QString, QString> SearchEngine::initializeCategoryNames()
-{
-    QHash<QString, QString> result;
-
-    result["all"] = QT_TRANSLATE_NOOP("SearchEngine", "All categories");
-    result["movies"] = QT_TRANSLATE_NOOP("SearchEngine", "Movies");
-    result["tv"] = QT_TRANSLATE_NOOP("SearchEngine", "TV shows");
-    result["music"] = QT_TRANSLATE_NOOP("SearchEngine", "Music");
-    result["games"] = QT_TRANSLATE_NOOP("SearchEngine", "Games");
-    result["anime"] = QT_TRANSLATE_NOOP("SearchEngine", "Anime");
-    result["software"] = QT_TRANSLATE_NOOP("SearchEngine", "Software");
-    result["pictures"] = QT_TRANSLATE_NOOP("SearchEngine", "Pictures");
-    result["books"] = QT_TRANSLATE_NOOP("SearchEngine", "Books");
-
-    return result;
-}
-
-PluginVersion SearchEngine::getPluginVersion(QString filePath)
+PluginVersion SearchPluginManager::getPluginVersion(QString filePath)
 {
     QFile plugin(filePath);
     if (!plugin.exists()) {
```

Comment:
You can remove the curly braces around this if/else.

Reply:
@glassez and this

Reply:
Additionally, we shouldn't produce subjective changes. If some code is correct but can be written in different but equivalent way, we shouldn't change it.

Reply:
I think the [CODING_GUIDELINES](https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md) would **strongly** disagree with that statement. Specifically, section 2: If blocks.

Reply:
BTW, in this particular case the block has multiple statements and I can't remove braces. Apparently, you didn't understand the diff correctly.

Reply:
>I think the CODING_GUIDELINES would strongly disagree with that statement. 

With what statement exactly?